BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx

```diff
@@ -27,7 +28,10 @@ class CancelBuildButton extends Component {
     this.setState({
       cancelling: true,
       showModal: false
-    });    
+    });
+
+    const parentPath = window.location.pathname.slice(0, window.location.pathname.lastIndexOf('/'));
```

Comment:
you can also do it one better and create a custom reusable history util module to store these types of functions, and import it where you need it.